app/locales/ru.json

```diff
@@ -67,6 +67,7 @@
     "event_history_subtitle": "Оцените собственные риски на основе информации от органов здравоохранения.",
     "event_history_title": "История возможных рисков",
     "filter_authorities_by_gps_history": "Отсортировать по местоположению",
+    "go_back": "Вернуться на предыдущую страницу",
```

Comment:
@alpita-masurkar does a one-off translation cause any issues for the translation flow?